andela/titan-dev-ah-frontend
# #165679900 Display Success/Error Message Respectively

#### What does this PR do?
- Refactor success/error message
#### Description of Task to be completed?
- Create a popup success/error message
#### How should this be manually tested?
- Clone the repo and check out the branch `ch-fix-success-error-message-165679900`
- Install all dependencies using `yarn` 
- Run tests by running `yarn test`
- Start your front-end and back-end server. 
- On the home page: click on the profile avatar, and click on the new story. 
- Create the story.  After creating a story, you should see a message of successful creation disappearing in 3 seconds. 

#### What are the relevant pivotal tracker stories?
#165679900
